ted_sws/notice_fetcher/adapters/ted_api_abc.py

```diff
@@ -3,6 +3,12 @@
 from typing import List
 
 
+class RequestAPI(abc.ABC):
```

Comment:
The value of this abstract class is to define the call methods. 
I would recommend defining those here.

Reply:
Done